README.md

```diff
@@ -7,12 +7,11 @@ and JavaScript that's *built* in HTML, CSS and JavaScript.
 What makes Brackets different from other web code editors?
 
 * **Tools shouldn't get in your way.** Instead of cluttering up your coding
-environment with lots of panels and icons, the Quick Edit UI in Brackets puts 
-context-specific code and tools inline.
+environment with lots of panels and icons, the Quick Edit UI in Brackets puts context-specific code and tools inline.
```

Comment:
Pointless change: concatenating two source lines to make one longer source line. No effect on the output.